Aztec/Classes/Public/TextKit/AztecTextAttachment.swift

```diff
@@ -40,22 +35,80 @@ public class AztecTextAttachment: NSTextAttachment
         super.init(coder: aDecoder)
     }
 
+    // MARK: - Origin calculation
+
+    func xPosition(forContainerWidth containerWidth: CGFloat) -> Int {
+
```

Comment:
As far as I know we don't have standards on spacing, which IMHO is kind of a good thing as it means we can use it to increase readability.